docs/src/pages/[platform]/theming/style-props/examples/BorderExamples.tsx

```diff
@@ -2,7 +2,16 @@ import { Button, useTheme } from '@aws-amplify/ui-react';
 
 // use any CSS value
 export const BorderStylePropExample = () => {
-  return <Button borderRadius="0.5rem">Border Styling Example</Button>;
+  return (
+    <Button
+      borderRadius="0.5rem"
+      borderStyle={'dashed'}
+      borderColor={'red'}
+      borderWidth={'medium'}
```

Comment:
Small update to some formatting

```suggestion
      borderStyle="dashed"
      borderColor="red"
      borderWidth="medium"
```